src/scripts/main.js

```diff
@@ -1,8 +1,28 @@
 'use strict';
 
 const people = require('./lib/people');
+const tableElement = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+for (const person of people) {
+  const trElement = document.createElement('tr');
+  const tdName = document.createElement('td');
+  const tdGender = document.createElement('td');
+  const tdBorn = document.createElement('td');
+  const tdDied = document.createElement('td');
+  const tdAge = document.createElement('td');
+  const tdCentury = document.createElement('td');
 
-// write your code here
+  tableElement.append(trElement);
+  tdName.textContent = `${person.name}`;
+  trElement.append(tdName);
+  tdGender.textContent = `${person.sex}`;
+  trElement.append(tdGender);
+  tdBorn.textContent = `${person.born}`;
+  trElement.append(tdBorn);
+  tdDied.textContent = `${person.died}`;
+  trElement.append(tdDied);
+  tdAge.textContent = `${person.died - person.born}`;
+  trElement.append(tdAge);
+  tdCentury.textContent = `${Math.ceil(person.died / 100)}`;
+  trElement.append(tdCentury);
```

Comment:
Looks too complicated, you can find only "tr" element and change it `innerHTML `and then append it to `tableElement `

Reply:
i don't understand your way of thinking